Blockzilla/BrowserViewController.swift

```diff
@@ -894,6 +900,12 @@ class BrowserViewController: UIViewController {
                              input: "]",
                              modifierFlags: .command,
                              propertyList: nil),
+                UIKeyCommand(title: UIConstants.strings.shareMenuFindInPage,
```

Comment:
This string re-use is ok now but I filed #2519 to improve that later in a separate PR.